pre-piksi-multi-launch-updates.md

```diff
@@ -0,0 +1,222 @@
+### SBP changes before shipping Piksi Multi
+
+There are a few changes we would like to make in the SBP protocol. These changes
+will be much more painful to make after we ship and customers are relying on
+the protocol to be largely static, so we'd like to come to a consensus on
+these and implement them before shipping Piksi Multi.
+
+These changes attempt to accomplish a few things:
+- Make messages more bandwidth efficient
+- Make user side logic of using messages more obvious
+- Remove state from the protocol, where possible
+
+### 1. Send solution messages at all times, regardless of fix status
+
+- Affects the following messages:
+   - GPS time
+   - DOPS (to add)
+   - Pos ECEF
+   - Pos LLH
+   - Baseline ECEF
+   - Baseline NED
+   - Vel ECEF
+   - Vel NED
+   - Baseline heading 
+- Update flags 
+   - Describe "fix mode" portion of flags field to be for each individual message only
+   - in all messages, 0 corresponds to "invalid"
+   - RTK Float and RTK Fixed flags have same numerical value across different messages where relevant
+   - No separate flags for simulation mode
+   - Only a minor release without a change to MSG_ID can be used to assign a meaning to reserved flags
+- Flags detail
+    - POS flags:
+       - Bits 2-0
+         - 000 Invalid
```

Comment:
For LLH messages, There is a horizontal error and vertical error field precise to  milliimeters
For the ECEF messages, there is just one "Error" term precise to milimeters